pad/sony/dualsense/config.json

```diff
@@ -0,0 +1,87 @@
+{
+    "name": "Sony Dualsense Wireless Controller",
+    "functions": {
```

Comment:
New elements "macro" and "turbo" were now added to "functions) (See https://github.com/Wolfy76700/ControllerDatabase/wiki/Controller-Definition#functions_macro and https://github.com/Wolfy76700/ControllerDatabase/wiki/Controller-Definition#functions_turbo)